src/services/phases/brla/index.tsx

```diff
@@ -0,0 +1,123 @@
+import Big from 'big.js';
+import { createPublicClient, http } from 'viem';
+import { moonbeam } from 'viem/chains';
+
+import { SIGNING_SERVICE_URL } from '../../../constants/constants';
+import { OfframpingState } from '../../offrampingFlow';
+import { fetchOfframpStatus } from '../../signingService';
+import Erc20ABI from '../../../../mooncontracts/Erc20ABI.json';
```

Comment:
Let's rename this to make it clear that it's the moonbeam specific one. 
```suggestion
import { default as MoonbeamErc20Abi } from '../../../../mooncontracts/Erc20ABI.json';
```

Reply:
I think they're the same as any EVM (they should), but okay.

Reply:
Kind of, yes. But if they are the same, you could also re-use the existing ERC20 abi that we have in `src/contracts/ERC20.ts`. I assumed you tried that and it didn't work. I saw that some fields differ slightly between the one we already have and the new one. If you didn't try it yet, maybe the existing ERC20 works for both Moonbeam and Pendulum.

Reply:
I missed that one! I'll try to replace it.